bigquery/google/cloud/bigquery/job.py

```diff
@@ -2475,115 +2475,268 @@ def __eq__(self, other):
 
 
 class QueryPlanEntry(object):
-    """Map a single entry in a query plan.
+    """QueryPlanEntry represents a single stage of a query execution plan.
 
-    :type name: str
-    :param name: name of the entry
+    See
+    https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs
+    for the underlying API representation within query statistics.
 
-    :type entry_id: int
-    :param entry_id: ID of the entry
+    """
 
-    :type wait_ratio_avg: float
-    :param wait_ratio_avg: average wait ratio
+    _PROPERTY_TO_API_FIELD = {
```

Comment:
Actually, I think we can drop this altogether, right? This sister dictionaries to `_PROPERTY_TO_API_FIELD` are used to convert from the update field mask to the JSON object, which isn't necessary for this read-only class.

Reply:
I'll go ahead and drop it.  I wrote it half to keep track of all the property methods I needed to author, as well as for possible testing.